tests/fsharp/Compiler/Warnings/PatternMatchingWarningTests.fs

```diff
@@ -0,0 +1,536 @@
+﻿namespace FSharp.Compiler.UnitTests
+
+open NUnit.Framework
+open FSharp.Compiler.SourceCodeServices
+
+[<TestFixture>]
+module PatternMatchingWarningTests =
+
+    [<Test>]
+    let ``Complete pattern match on byte should not trigger FS0025 warning`` () =
+        CompilerAssert.CompileExe
```

Comment:
`Pass` would be better here, since we're not really interested in running an executable.

Reply:
Changed to Pass.